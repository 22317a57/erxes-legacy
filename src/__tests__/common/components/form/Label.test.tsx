import { shallow } from 'enzyme';
import React from 'react';

import Label from '../../../../modules/common/components/form/Label';

describe('Label component', () => {
  const defaultProps = {
    children: false
  };

  test('renders Label successfully', () => {
    shallow(<Label {...defaultProps} />);
  });
});
