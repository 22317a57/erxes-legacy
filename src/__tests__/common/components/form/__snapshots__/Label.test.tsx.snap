// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component snapshot matches 1`] = `
<label
  className="sc-iwsKbI dHozau"
/>
`;
